Item(by='dodobirdlord', descendants=None, kids=None, score=None, time=1606358463, title=None, item_type='comment', url=None, parent=25213828, text='In early 2019 CloudWatch had a major outage that was particularly nasty, where instead of just outright failing to report metrics it reported a small percentage of metrics. As a result a lot of autoscaling groups and DynamoDB tables that were theoretically supposed to avoid scaling in during a metric outage still scaled in, because they saw it as a 90+% traffic reduction rather than a metric outage.')